src/components/roverModel/RoverModel.js

```diff
@@ -221,6 +219,18 @@ function Hand() {
   );
 }
 
+function ArmSegment({length}) {
+  return <>
+    <Box args={[6, 6, length]}
```

Comment:
To keep formatting consistent with the rest of the file, it might be worth reformatting this as:
```
<Box
  args={[6, 6, length]}
  position={[0, 0, length/2]}
  material-color="#6C6C76"
/>
```